Tests/AdyenTests/Adyen Tests/Utilities/ComponentManagerTests.swift

```diff
@@ -58,10 +59,10 @@ class ComponentManagerTests: XCTestCase {
                                    order: nil)
 
         XCTAssertEqual(sut.storedComponents.count, 4)
-        XCTAssertEqual(sut.regularComponents.count, 15)
+        XCTAssertEqual(sut.regularComponents.count, 16)
```

Comment:
this a a bit brittle. can we do
```
XCTAssertEqual(sut.regularComponents.count, dictionary["paymentMethods"]?.count)
```

Reply:
actually `dictionary["paymentMethods"]?.count` is not 16, some PM's are not supported on mobile, so we need to check they are filtered out.